tests/fixtures/Cargo.lock

```diff
@@ -0,0 +1,5585 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
+version = 3
```

Comment:
Why do we have this plus a version 3 lockfile? Shouldn't one of the two be sufficient?

Reply:
we needed an example of using the URL as the version, but I can move the longer lockfile into the v3 spot.